INSTALLATION.rst

```diff
@@ -66,14 +66,19 @@ Recommended Method
 
     .. attention::
 
-       Note, this does not work on Windows. On that platform, you should be able to do::
+       Note, this does not work on Windows x86, as conda-forge_ no longer
+       supports that platform.  For Python 2.7.x, you should be able to do::
 
-        conda create --name <MYFIPYENV> --channel guyer --channel conda-forge python numpy scipy matplotlib mayavi
-        activate <MYFIPYENV>
-        pip install fipy
+        conda create --name <MYFIPYENV> --channel conda-forge python=2.7 numpy scipy matplotlib pysparse mayavi weave
 
-       There are presently no conda packages of any solver suite but scipy available for Windows.
+       and for Python 3.x, you should be able to do::
 
+        conda create --name <MYFIPYENV> --channel conda-forge python=3.6 numpy scipy matplotlib pysparse
+
+       followed, for either, by::
+
+        activate <MYFIPYENV>
+        pip install fipy
```

Comment:
Does this actually work?

Reply:
Might could? I intend to address this with #565.